test/GitHub.TeamFoundation.UnitTests/VSGitExtTests.cs

```diff
@@ -17,7 +17,7 @@
 
 public class VSGitExtTests
 {
-    public class TheConstructor : TestBaseClass
+    public class TheConstructor
```

Comment:
Why is this inheritance changing?

Reply:
Ummm... I can't remember. I'll put this back.